pkg/log/log.go

```diff
@@ -12,43 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package log provides the canonical logging functionality used by Go-based
```

Comment:
why is this being deleted in this PR ?

Reply:
Turns out this comment was also verbatim in config.go. I did some sloppy copy&paste when creating the files in the first place.